src/libraries/System.Security.Cryptography/tests/SlhDsaOpenSslTests.cs

```diff
@@ -75,8 +75,7 @@ public void SlhDsaOpenSsl_DuplicateKeyHandleLifetime()
 
         private static void VerifyInstanceIsUsable(SlhDsaOpenSsl slhDsa)
         {
-            byte[] secretKey = new byte[slhDsa.Algorithm.SecretKeySizeInBytes];
-            Assert.Equal(slhDsa.Algorithm.SecretKeySizeInBytes, slhDsa.ExportSlhDsaSecretKey(secretKey)); // does not throw
+            byte[] secretKey = slhDsa.ExportSlhDsaSecretKey();
```

Comment:
Since nothing uses `secretKey` we usually leave a comment explaining why this is here:

```suggestion
            byte[] secretKey = slhDsa.ExportSlhDsaSecretKey(); // Assert.NoThrow
```

Reply:
Reverted the unrelated change.